certify-service/src/main/java/io/mosip/certify/config/TemplateConfig.java

```diff
@@ -66,12 +66,12 @@ public void run(String... args) throws Exception {
             throw new CertifyException("Missing well known config");
         }
 
-        List<SvgRenderTemplate> svgRenderTemplateList = svgRenderTemplateRepository.findAll();
+//        List<SvgRenderTemplate> svgRenderTemplateList = svgRenderTemplateRepository.findAll();
 
-        if(svgRenderTemplateList.isEmpty()) {
+//        if(svgRenderTemplateList.isEmpty()) {
             svgTemplateMap.forEach((key, value) -> {
                 SvgRenderTemplate svgRenderTemplate = new SvgRenderTemplate();
-                svgRenderTemplate.setId(UUID.randomUUID());
+                svgRenderTemplate.setId(key);
                 if(domainUrl.startsWith("https")) {
```

Comment:
starts with `http` might be a good check @Piyush7034

Reply:
Changed the above check with http